Documentation/openshift-issues.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Common Issues on OpenShift
+weight: 10620
+indent: true
+---
+
+# Common Issues on OpenShift
+
+## Monitoring Ceph Cluster on OpenShift
+
+OpenShift Console uses OpenShift Prometheus for monitoring and populating data in Storage Dashboard. Following troubleshooting guide assumes that it is allowed to monitor the Ceph Cluster.
+
+1. Check if Ceph Cluster monitoring is enabled
+
+    `$ kubectl get cephcluster rook-ceph -n rook-ceph -oyaml`
+
+    ```YAML
+    apiVersion: ceph.rook.io/v1
+    kind: CephCluster
+    metadata:
+    name: rook-ceph
+    namespace: rook-ceph
+    [...]
+    spec:
+    [...]
+    monitoring:
+        enabled: true
+        rulesNamespace: "rook-ceph"
+    [...]
+    ```
+
+    > **Note:** Follow [ceph-monitoring/prometheus-alerts](https://github.com/rook/rook/blob/master/Documentation/ceph-monitoring.md#prometheus-alerts) to enable Ceph Cluster monitoring.
```

Comment:
Step 1 should be to enable monitoring and we just need to refer them to the other doc, right? I think that doc already mentions everything about this CephCluster setting. No need to include instructions here to `get CephCluster`.

Reply:
Agreed.